static/refcache.json

```
@@ -27,6 +27,10 @@
     "StatusCode": 200,
     "LastSeen": "2024-08-09T10:46:17.075695-04:00"
   },
+  "https://actix.rs/": {
+    "StatusCode": 502,
```

Server error, but the link is valid, so let's manually update the status to reflect that:

```suggestion
    "StatusCode": 206,
```